### ISSUE TITLE
No support for 'oshcc -static'

### ISSUE THREAD

**Issue by davidozog** [Sandia-OpenSHMEM/SOS#711]
Similar to `mpicc -static`, which prints a static library link line (I believe).

**Comment by jdinan** [comment]
Minor wibble -- this is a feature request not a bug report.

**Comment by davidozog** [comment]
Yup, I thought we had a label for feature requests.  "Enhancement"?
